pkg/azurefile/azurefile.go

```diff
@@ -300,6 +303,7 @@ func NewDriver(options *DriverOptions) *Driver {
 	driver.volLockMap = newLockMap()
 	driver.subnetLockMap = newLockMap()
 	driver.volumeLocks = newVolumeLocks()
+	driver.azcopy = &fileutil.Azcopy{}
```

Comment:
if you define `azcopy fileutil.Azcopy`, then this line is not necessary?

Reply:
we need to set a value for `d.azcopy` but not nil for L120-122 in `util.go`
```
	if ac.ExecCmd == nil {
		ac.ExecCmd = &ExecCommand{}
	}
```